pootle/core/cache.py

```diff
@@ -43,3 +47,14 @@ def make_key(*args, **kwargs):
     return ':'.join([
         '%s=%s' % (k, v) for k, v in sorted(kwargs.iteritems())
     ])
+
+
+def get_cache(cache=None):
+    """Return a `cache` object named `cache` in none is specified or if the
```

Comment:
*"a `cache` object named `cache`"* doesn't make much sense — you just want to return the default cache if none has been specified.

There's also a typo in the sentence, *in* should be *if*.